packages/graphql-cli-graphback-plugin/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{ 
```

Comment:
This `tsconfig.json` is not extending the top level one. It should probably be replaced with the exact same `tsconfig.json` you would find in the `graphback` and `graphback-cli` packages